workflow/rocoto/workflow_xml.py

```diff
@@ -8,6 +8,7 @@
 from applications.applications import AppConfig
 from rocoto.workflow_tasks import get_wf_tasks
 import rocoto.rocoto as rocoto
+import numpy as np
```

Comment:
Why `np`?
`strings` is a `list`.  `strings.append` will get you the result.

Reply:
We test to remove "np".

Reply:
Remove "np", and use straings.extend works.